docs/develop/create-addin-commands.md

```diff
@@ -406,29 +327,32 @@ The following shows an example of how to use the **\<Resources\>** element. Each
 </Resources>
 ```
 
-|Resource|Description|
-|:-----|:-----|
-|**\<Images\>**/ **\<Image\>** <br/> | Provides the HTTPS URL to an image file. Each image must define the three required image sizes: <br/>  16×16 <br/>  32×32 <br/>  80×80 <br/>  The following image sizes are also supported, but not required: <br/>  20×20 <br/>  24×24 <br/>  40×40 <br/>  48×48 <br/>  64×64 <br/> |
-|**\<Urls\>**/ **\<Url\>** <br/> |Provides an HTTPS URL location. A URL can be a maximum of 2048 characters.  <br/> |
-|**\<ShortStrings\>**/ **\<String\>** <br/> |The text for **\<Label\>** and **\<Title\>** elements. Each **\<String\>** contains a maximum of 125 characters. <br/> |
-|**\<LongStrings\>**/ **\<String\>** <br/> |The text for **\<Tooltip\>** and **\<Description\>** elements. Each **\<String\>** contains a maximum of 250 characters. <br/> |
-
 > [!NOTE]
 > You must use Secure Sockets Layer (SSL) for all URLs in the **\<Image\>** and **\<Url\>** elements.
 
-### Tab values for default Office app ribbon tabs
+## Add-in commands in a Teams Manifest (developer preview)
+
+Add-in commands are declared with the "extensions.runtimes" and "extensions.ribbons" properties. These properties specify many things for the add-in, such as the application, types of controls to add to the ribbon, the text, the icons, and any associated functions.
+
+When an add-in needs to provide status updates, such as progress indicators or error messages, it must do so through the [notification APIs](/javascript/api/outlook/office.notificationmessages). The processing for the notifications must also be defined in a separate HTML file that is specified in the "runtimes.code.page" property of the manifest.
```

Comment:
This section needs to make clear that the JSON manifest is only supported in Outlook. Also, we should link to https://learn.microsoft.com/en-us/office/dev/add-ins/develop/json-manifest-overview